File: api/src/main/java/net/blueshell/api/repository/EventFeedbackRepository.java
package net.blueshell.api.repository;

import net.blueshell.api.base.BaseRepository;
import net.blueshell.api.model.EventFeedback;
import org.springframework.stereotype.Repository;

@Repository
public interface EventFeedbackRepository extends BaseRepository<EventFeedback, Long> {
}

File: api/src/main/kotlin/net/blueshell/api/feature/event/repository/EventFeedbackRepository.kt
package net.blueshell.api.feature.event.repository

import net.blueshell.api.feature.event.model.EventFeedback
import net.blueshell.api.shared.repository.BaseRepository
import org.springframework.stereotype.Repository

@Repository
interface EventFeedbackRepository : BaseRepository<EventFeedback, Long>
